### DISCUSSION TITLE
Copilot: Deletion of all the Code Snippet Data: My legal request remains unanswered for days

### DISCUSSION THREAD

**Discussion by arthurpetit** [community/community#42827] [Copilot Conversations]
### Select Topic Area

General

### Body

I have requested the deletion of all copilot Code Snippets Data associated with my identity and this legal request remains unanswered for days now.

**Comment by RishatM** [comment]
Just curious was this resolved for you?

**Comment by arthurpetit** [comment]
It seems that it does. I hope so.



On Fri 24 Mar 2023 at 17:25, RishatM ***@***.***> wrote:

> Just curious was this resolved for you?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/42827#discussioncomment-5420554>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAWQZAIKI7EEOOY76JDRUUDW5XDF3ANCNFSM6AAAAAATL5CU5I>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Arthur Petit
